0128-energy-subnetwork.md

```diff
@@ -95,15 +96,14 @@ We propose using one pool for buffering issued baseline rewards:
 
 **Network growth pool** acts as a resource for growing the network using direct incentives to DER hosts. This pool will receive tokens from the baseline DER host rewards inversely proportional to the total Energy subnetwork DC burn as long as this is less than $100 000 USD. The proportion allocated to the pool is calculated by:
 
-Network&nbsp;Growth&nbsp;Pool&nbsp;Proportion =&nbsp;0.5&nbsp;&minus;&nbsp;min(0.5,&nbsp;0.5&nbsp;*&nbsp;Total&nbsp;Energy&nbsp;Subnetwork&nbsp;DC&nbsp;burn&nbsp;/&nbsp;1e10)
+Network&nbsp;Growth&nbsp;Pool&nbsp;Proportion =&nbsp;0.5&nbsp;&minus;&nbsp;min(0.5,&nbsp;0.5&nbsp;\*&nbsp;Total&nbsp;Energy&nbsp;Subnetwork&nbsp;DC&nbsp;burn&nbsp;/&nbsp;1e10)
```

Comment:
Perhaps we can do this in LaTeX to clean up the syntax.

---
$\text{Network Growth Pool Proportion}= 0.5 - \min\left(0.5, 0.5 \times \frac{\text{Total Energy Subnetwork DC burn}}{1e10}\right)$

Reply:
```latex
$\text{Network Growth Pool Proportion} = 0.5 - \min\left(0.5, 0.5 \times \frac{\text{Total Energy Subnetwork DC burn}}{1e10}\right)$
```